testsystem/test_objects.proto

```
@@ -0,0 +1,16 @@
+    required string description = 3; // Optional description of the test script
+    repeated string steps = 4; // List of steps in the test script
+    map<string, string> metadata = 5; // Additional metadata for the test script
+    optional bool is_active = 6; // Indicates if the script is currently active or deprecated
+}
+
+message TestSuite {
+    required string name = 1; // Name of the test suite
+    repeated TestScript scripts = 2; // List of test scripts in the suite
+    required string description = 3; // Optional description of the test suite
+    map<string, string> metadata = 4; // Additional metadata for the test suite
+    optional repeated TestSuite sub_suites = 5; // Nested test suites
```

Buf (1.54.0)

syntax error: expecting ';'

(COMPILE)